integration/Substra/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Other
+local-worker/
+experiment_summaries/*
+!experiment_summaries/.gitkeep
+*.zip
```

Comment:
what is the `empty` folder used for?

Reply:
It is to point a folder for the datasamples, as it is a required arg but we have no data to load in a FLamby setup